Assets/_BForBoss/Scripts/Managers/WorldManager.cs

```diff
@@ -96,10 +111,16 @@ private void HandleStateChange(State newState)
                 }
                 case State.Play:
                 {
+                    if (Time.timeScale != 1.0f)
```

Comment:
Do you need this check?

Reply:
Yeah I was thinking of not adding the check There is no way that State.PreGame or that State.EndRace will have a different timescale?

Reply:
I can remove this for now